Shared/Characters/PlayerBase.gd

```diff
@@ -65,10 +69,11 @@ func apply_input(movement_vector: Vector3, rotation_vector: Vector3, buttons: in
 	# Apply acceleration to velocity (important: after lerp)
 	self.velocity += acceleration * delta
 	var _collision_velocity = _kb.move_and_slide(velocity)
-	
+
 	# Trigger all actions with base
-	.trigger_actions(buttons)
-	
+	if aim_mode == false:
+		.trigger_actions(buttons)
```

Comment:
As this is inside `PlayerBase` and the `Enemy` inherits from it, did you test if enemy actions are still triggered locally? Didn't see `aim_mode` being send to server and other client